webapp/hooks/usePrintView.ts

```diff
@@ -0,0 +1,4 @@
+import { useSelector } from 'react-redux'
+import * as AppState from '@webapp/store/app/state'
+
+export default () => useSelector((state) => [AppState.isPrintView(state), AppState.isPrintOnlyTablesView(state)])
```

Comment:
these hooks belong to store/app.
Plus, AppState should not be used anymore to get data.
AppState must be only an interface.
our state is typed now